medusa/providers/torrent/html/limetorrents.py

```diff
@@ -57,6 +57,7 @@ def __init__(self):
             'update': urljoin(self.url, '/post/updatestats.php'),
             'search': urljoin(self.url, '/search/tv/{query}/'),
             'rss': urljoin(self.url, '/browse-torrents/TV-shows/date/{page}/'),
+            'tv': urljoin(self.url, '/browse-torrents/TV-shows/'),
```

Comment:
TV or rss is just a convention we use? We have lots of 'rss' URLs that work perfectly.